blitzstudios/sleeper-mini
# Jasonl/feat/fetch

### Description
Updating sleeper-mini-core version to 1.2
Adding separate demo project for fetching from APIs through the context.

### How To Test
1. Connect to Sleeper development app
2. Modify app.json to select appropriate demo
3. Check that requested data is coming back properly.
